cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less

```diff
@@ -1 +1,298 @@
+  margin: 4px;
+}
+
+.c-item-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.c-item {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.c-step-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.c-step-img,
+.c-step-title {
+  display: block;
+  margin: auto;
+}
+.c-step-img {
+  width: 75%;
+}
+
+.c-step-title {
+  margin-top: 6px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  text-align: center;
+}
+
+.c-nav-buttons {
+  margin-top: 6px;
+  padding: 4px;
+}
+
+.c-nav-buttons div {
+  margin-top: 6px;
+  margin-bottom: 6px;
+}
+
+.c-calendar-img {
+  width: 80%;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.c-item-list,
+.c-item-list-two {
+  width: 80%;
+  margin: 0 auto;
+  padding-top: 6px;
+}
+
+.c-item-list-four {
+  margin: 10px auto;
+  padding-top: 6px;
+}
+
+.green {
+  color: green;
+}
+
+.red {
+  color: red;
+}
+
+.c-item-list-two li {
+  display: flex;
+  flex-direction: row;
+
+  div {
+    padding-left: 4px;
+    padding-right: 4px;
+    width: 50%;
+  }
+}
+
+.c-item-list-four li {
+  display: flex;
+  flex-direction: row;
+  font-size: 14px;
+  padding-top: 4px;
+  padding-bottom: 6px;
+
+  div {
+    padding-left: 4px;
+    padding-right: 4px;
+    width: 22%;
+    border-bottom: 1px solid;
+  }
+}
+
+.c-input-img {
+  width: 60%;
+  margin: 0 auto;
+  img {
+    width: 100%;
+  }
+}
+
+.m-hero_wrapper {
+  padding-top: 0.8em;
+  padding-bottom: 0;
+}
+
+.c-month-nav-links {
+  display: flex;
+  flex-direction: row;
+  margin: 10px auto;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  border-top: 1px solid;
+  border-bottom: 1px solid;
+
+  div {
+    width: 48%;
+    text-align: center;
+  }
+}
+
+.c-strategies-banner {
+  background-color: #f6f8f9;
+  border: 1px solid lightgrey;
+  padding: 6px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.c-strategies-banner-container {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 4px;
+}
+
+.c-click-to-view-link {
+  text-align: center;
+  border-top: 1px solid;
+  border-bottom: 1px solid;
+  margin-top: 6px;
+  margin-bottom: 12px;
+}
+
+.c-input-summary {
+  width: 70%;
+  margin: 0 auto;
+  margin-bottom: 6px;
+}
+
+.c-star-icon {
+  height: 25px;
+}
+
+.c-strategies {
+  ul {
+    list-style-type: none;
+    margin-top: 10px;
+  }
+  li {
+    background-color: #f6f8f9;
+    border: 1px solid lightgrey;
+    padding: 6px;
+    margin-top: 5px;
+    margin-bottom: 10px;
+  }
+}
+
+.c-category-input-container {
+  margin-bottom: 30px;
+}
+
+.c-input {
+  height: 40px;
+  width: 75%;
+  margin: 0 auto;
+}
+
+.c-category-input {
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  background-color: #d3daeb;
+  border: 2px solid #556b90;
+}
+
+.c-radio-input-fieldset {
+  border: 0;
+  width: 50%;
+  margin: auto;
+}
+
+.c-cfevent-list-container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.c-cfevent-list {
+  display: flex;
+  flex-direction: row;
+  div {
```

Comment:
In general it's not great practice to use tag names in selectors if you have control over the markup. Classes are much faster to look up.